### PR TITLE
Refactor Readme and add a contributing doc

### PR DESCRIPTION
**Pull request by xhlulu**
The current readme contains a lot of internal information that might not be useful for most users visiting the repository, and the starting paragraph is a bit hard to understand.

I made those small changes (click here for [readme](https://github.com/plotly/dash-sample-apps/blob/3e426bdc79dce26d4aef127e68d90ab2ab1684b0/README.md)):
* Rewrite the starting paragraph to be clearer
* Move up the zip download instructions to be the first thing any user sees
* Everything else to a "Contributing.md" file

Let me know if those changes make sense!

Closes https://github.com/plotly/dash-sample-apps/issues/633

### CODE REVIEW

**Comment by xhlulu** [comment]
@plotly/marketing @alexcjohnson Any thoughts on this? This will probably be a pretty impactful change since sample apps gets 40k visits per month

File: README.md [hunk 1]
```diff
@@ -39,172 +21,15 @@ then run the app:
...
-```
-# once your branch is ready, make a PR into master!
-
-PR has two checkers.
-1. make sure your code passed the black linter
-2. make sure your project is deployed on dns playground
-```
+## Contributing
 
+To contribute to this repository, please see the [contributing document](CONTRIBUTING.md).
```
Comment (alexcjohnson) [inline README.md:35]:
```suggestion
To contribute to this repository, please see the [contributing guide](CONTRIBUTING.md).
```

File: README.md [hunk 2]
```diff
@@ -39,172 +21,15 @@ then run the app:
...
-2. Ensure that the file containing your app code is named `app.R`.
-
-3. The `Procfile` should contain 
-
-```
-web: R -f /app/app.R
-```
-
-4. Routing and request pathname prefixes should be set. One approach might be to include
+## Cloning a repository
```
Comment (alexcjohnson) [inline README.md:24]:
```suggestion
## Cloning this whole repository
```

**Review by alexcjohnson — APPROVED** [review]
💃 I like it!

**Pull request merged**